app/screens/select_team/__snapshots__/select_team.test.js.snap

```diff
@@ -143,3 +143,37 @@ exports[`SelectTeam should match snapshot for teams 1`] = `
   />
 </View>
 `;
+
+exports[`SelectTeam should match snapshot when user is a guest 1`] = `
+<View
+  style={
+    Object {
+      "backgroundColor": "#ffffff",
+      "flex": 1,
+    }
+  }
+>
+  <Connect(StatusBar) />
+  <View
+    style={
+      Object {
+        "alignItems": "center",
+        "flexDirection": "row",
+        "marginHorizontal": 16,
+        "marginTop": 20,
+      }
+    }
+  >
+    <FormattedText
+      defaultMessage="You don't belong to any team and as a guest you can't join any team by yourself, you need to be invited or added by som  eone else."
```

Comment:
@jespino does the message need to be updated here as well?

Reply:
Oops, yes